pkg/apis/ignite/v1alpha1/conversion.go

```diff
@@ -84,3 +84,103 @@ func Convert_v1alpha1_OCIImageSource_To_ignite_OCIImageSource(in *OCIImageSource
 	out.ID, err = meta.ParseOCIContentID(contentRef)
 	return
 }
+
+// Convert_ignite_ImageSpec_To_v1alpha1_ImageSpec calls the autogenerated conversion function along with custom conversion logic
+func Convert_ignite_ImageSpec_To_v1alpha1_ImageSpec(in *ignite.ImageSpec, out *ImageSpec, s conversion.Scope) error {
+	if err := autoConvert_ignite_ImageSpec_To_v1alpha1_ImageSpec(in, out, s); err != nil {
+		return err
+	}
+
+	// Convert between the old and new OCI reference format
```

Comment:
maybe create a `Convert_v1alpha1_OCIClaim_To_ignite_OCI(in *OCIClaim, out *meta.OCIImageRef) error` and vice-versa which you can call in all these higher-context functions?

Reply:
Done in 9d6ab9e.